### PR TITLE
Setup module packaging

### PR DESCRIPTION
**Pull request by jakobmoellersap**

### CODE REVIEW

File: operator/config/samples/component-integration-installed/moduletemplates/example/blobs/sha256:9d37fb366e4371b44c94496f3db276bd6f3a1019c9d44077b689c68ec6486c1e [hunk 1]
```diff
@@ -0,0 +1,7 @@
+configs:
+  - name: some-random-chart-name
+    clientConfig: "CreateNamespace=true,Namespace=jet-cert"
+    overrides: "x=1"
+  - name: some-other-random-chart-name
+    clientConfig: "Namespace=my-amazing-ns"
+    overrides: "x=1"
```
Comment (adityabhatia) [inline operator/config/samples/component-integration-installed/moduletemplates/example/blobs/sha256:9d37fb366e4371b44c94496f3db276bd6f3a1019c9d44077b689c68ec6486c1e:7]: we do not need the blobs folder or?

File: operator/controllers/kyma_controller.go [hunk 2]
```diff
@@ -252,15 +276,15 @@ func (r *KymaReconciler) HandleDeletingState(ctx context.Context, kyma *operator
 
 	if kyma.Spec.Sync.Enabled {
 		if err := remote.RemoveFinalizerFromRemoteKyma(ctx, r, kyma); client.IgnoreNotFound(err) != nil {
-			return false, err
+			return false, errwrap.Wrap(err, "error while trying to remove finalizer from remote")
```
Comment (adityabhatia) [inline operator/controllers/kyma_controller.go:279]: `errors.Wrap()` / `errors.Wrapf()` is obsolete. we could use `fmt.Errorf("your error string %w", err)` or `errors.Errorf()` for preserving strack trace

**Review by adityabhatia — CHANGES_REQUESTED** [review]
Couldn't check code in detail tbh, but works great 🚀 
One minor change required though

File: operator/config/samples/component-integration-installed/moduletemplates/example/blobs/sha256:9d37fb366e4371b44c94496f3db276bd6f3a1019c9d44077b689c68ec6486c1e (hunk 1 above)
Comment (jakobmoellersap) [reply operator/config/samples/component-integration-installed/moduletemplates/example/blobs/sha256:9d37fb366e4371b44c94496f3db276bd6f3a1019c9d44077b689c68ec6486c1e:7]: blobs are generated by the component-cli for the archive, so nothing from me

File: operator/controllers/kyma_controller.go (hunk 2 above)
Comment (jakobmoellersap) [reply operator/controllers/kyma_controller.go]: Fixed it by replacing all Wraps with fmt.Errorf (as this is also used in the k8s libs)

File: operator/controllers/kyma_controller.go [hunk 3]
```diff
@@ -354,7 +383,7 @@ func (r *KymaReconciler) CreateOrUpdateModules(ctx context.Context, kyma *operat
```
[the remark this answers is not in the document]
Comment (jakobmoellersap) [reply operator/controllers/kyma_controller.go:396]: I would think so templates logic still apply

**Review by adityabhatia — APPROVED** [review]

**Pull request merged**